### DISCUSSION TITLE
how can I get response headers, statusCode, etc, inside buffer?

### DISCUSSION THREAD

**Discussion by tednaaa** [mistweaverco/kulala.nvim#156] [Q&A] [answered]
sorry, for noob question, I checked docs, didn't found that case, is it possible to implement somehow?

<img width="760" alt="image" src="https://github.com/user-attachments/assets/b7390b39-694f-40ef-afc9-c9b1d5c7b524">

**Comment by gorillamoe** [comment, marked as answer]
Either by setting `default_view` to `headers_body` https://kulala.mwco.app/docs/getting-started/configuration-options#default_view

Or if you want to keep the default view, you can toggle body and headers with `toggle_view()` https://kulala.mwco.app/docs/usage/public-methods#toggle_view.
